common/resources.yaml

```diff
@@ -60,10 +60,12 @@ groups:
        - name: object_id
          dtype: text
          doc: The UUID for the object.
+       - name: relative_path
+         dtype: text
+         doc: The relative path of the Container.
```

Comment:
```suggestion
         doc: The relative path from the container with the object_id to the dataset or attribute
           with the value(s) that is associated with an external resource. This can be an empty 
           string if the container is a dataset and contains the value(s) that is associated 
           with an external resource.
```